lib/generators/solid_cache/install/install_generator.rb

```diff
@@ -51,7 +51,7 @@ def sqlite_database_config_with_cache
     end
 
     def app_name_from_production_database_name
-      database_yml.read.scan(/database: (\w+)_production/).flatten.first
+      database_yml.read.scan(/database: (\w+)_production/).flatten.first || "app"
```

Comment:
Rather than falling back to "app" directly, we could be using the Rails.application.name to get a fallback value.

```suggestion
database_yml.read.scan(/database: (\w+)_production/).flatten.first || (defined?(Rails) ? Rails.application.name.underscore : "app")
```

Reply:
Good call, thanks!